pkgs/top-level/all-packages.nix

```diff
@@ -20801,9 +20786,7 @@ in
   # Realtime kernel packages.
   linuxPackages-rt_5_4 = linuxPackagesFor pkgs.linux-rt_5_4;
   linuxPackages-rt_5_10 = linuxPackagesFor pkgs.linux-rt_5_10;
-  linuxPackages-rt_5_11 = linuxPackagesFor pkgs.linux-rt_5_11;
   linuxPackages-rt = linuxPackages-rt_5_4;
-  linuxPackages-rt_latest = linuxPackages-rt_5_11;
   linux-rt = linuxPackages-rt.kernel;
   linux-rt_latest = linuxPackages-rt_latest.kernel;
```

Comment:
While `linuxPackages-rt_5_11` is going to be removed alongside 5.11 I don't think `linuxPackages-rt_latest` should be removed, it should be repointed to the latest version either 5.10 or 5.12?

Reply:
Another possibility could be to delay 5.11 removal until linux-rt-5.12 gets released

Reply:
OMG! Did I do this? 😊 Let me fix this quickly before anyone noticed 🙂  Thanks!

Reply:
Should be fixed now. Was in a hurry yesterday. Thanks for the review @fabianhjr!

Reply:
You are welcome, thanks for pushing the Kernel Updates along :3